docker/build_whl.dockerfile

```diff
@@ -45,8 +45,8 @@ COPY . .
 
 CMD echo '--- Installing JS dependencies' && \
     yarn install --pure-lockfile && \
-    echo `--- Making whl` && \
```

Comment:
This is just to fix the error `/bin/sh: 1: ---: not found` when running the dockerfile